sql/civicrm_data/civicrm_job.sqldata.php

```diff
@@ -41,42 +40,45 @@
     [
       'run_frequency' => 'Daily',
       'name' => 'Process Pledges',
-      'description' => 'Updates pledge records and sends out reminders',
+      'description' => 'Updates pledge records and sends out reminders. Use parameters: send_reminders=[1 or 0], use 1 to send payment reminders',
       'api_entity' => 'Job',
       'api_action' => 'process_pledge',
-      'parameters' => 'send_reminders=[1 or 0] optional- 1 to send payment reminders',
+      'parameters' => 'send_reminders=0',
     ],
     [
       'run_frequency' => 'Daily',
       'name' => 'Geocode and Parse Addresses',
-      'description' => 'Retrieves geocodes (lat and long) and / or parses street addresses (populates street number, street name, etc.)',
+      'description' => 'Geocodes and/or parses street addresses. Use parameters: geocoding=[1 or 0] required
+parse=[1 or 0] required
+start=[Starting Contact ID]
+end=[Ending Contact ID]
+throttle=[1 or 0] - 1 adds five second sleep',
       'api_entity' => 'Job',
       'api_action' => 'geocode',
-      'parameters' => 'geocoding=[1 or 0] required
-parse=[1 or 0] required
-start=[contact ID] optional-begin with this contact ID
-end=[contact ID] optional-process contacts with IDs less than this
-throttle=[1 or 0] optional-1 adds five second sleep',
+      'parameters' => 'geocoding=1
+      parse=0
+      throttle=0',
```

Comment:
I think the lack of leading spaces was actually intentional here as each param is supposed to go at the start of a newline.

Reply:
Yes that's correct.